packages/webhooks/constants.js

```diff
@@ -1,5 +1,8 @@
 const DEFAULT_MAX_PACK_SIZE = 100
+// TODO(DOMA-4570) Correct this value based on received stats, when actual skip-send logic will be implemented
+const UNAVAILABILITY_THRESHOLD = 10
```

Comment:
Why not to get this value from environment? And if is not provided -> get this constant. It could be easier to change it in the future

Reply:
Since it's not dynamic story...it's a const. I've added mechanism of blocking hooks right now. But to set const we need to get real-usage data...to give fair const value based on real scenarios

Reply:
I mean not at this place) May be at place of this constant usage?

Reply:
Added optional parameter to config